docs/02-deployment-guide.md

```diff
@@ -125,73 +117,62 @@ sleeper environment destroy MyEnvironment "*-BuildEC2"
 
 Parameters after the environment ID will be passed to a `cdk destroy` command.
 
-## Building
-
-The first step is to build Sleeper. We have a section in our
-[developer guide](./09-dev-guide.md#Building) for how to do that but the easiest way is:
-
-```bash
-./scripts/build/build.sh
-```
-
 ## Deployment
 
 There are two ways to deploy Sleeper: you can use the automated scripts or a more manual approach.
 
 ### Automated Deployment
 
-The automated script uses template files to provide a default
-configuration for Sleeper. It also deploys only one table
-into Sleeper with the schema provided in these template files.
-You can find the template files [here](../scripts/templates).
+The automated deployment uses template files to provide a default configuration for Sleeper. It also deploys only one
+table into Sleeper with the schema provided in these template files. You can find the template
+files [here](../scripts/templates).
 
-It is recommended that you change these templates to configure Sleeper
-in the way that you want before you run the automated script. At
-the very least you will want to change the schema.template and
-tags.template files. See the Configuration section below
-for further details.
+It is recommended that you change these templates to configure Sleeper in the way that you want before you run the
+automated script. At the very least you will want to change the schema.template and tags.template files. See the
+Configuration section below for further details.
 
-Note that anything property in the templates with "changeme"
-as its value will be overwritten automatically by the scripts.
+Note that anything property in the templates with "changeme" as its value will be overwritten automatically.
```

Comment:
Looks like this should be `Note that any property in the templates with "changeme" as its value ...`